persistence/pom.xml

```diff
@@ -62,9 +62,9 @@
             <version>6.5.0</version>
         </dependency>
         <dependency>
-            <groupId>com.arangodb</groupId>
-            <artifactId>velocypack-module-scala</artifactId>
-            <version>1.1.0</version>
+            <groupId>com.github.wajda.com.arangodb</groupId>
+            <artifactId>velocypack-module-scala_${scala.compat.version}</artifactId>
+            <version>1.1.0.1</version>
```

Comment:
It looks like, they merged your changes. https://github.com/arangodb/java-velocypack-module-scala/releases/tag/v1.2.0

So we can use the original version again, right?

Reply:
I'm still waiting for it to appear in Maven Central. It's not there yet - https://repo1.maven.org/maven2/com/arangodb/velocypack-module-scala_2.12/1.2.0/velocypack-module-scala_2.12-1.2.0.pom